packages/contracts-bedrock/test/setup/ForkLive.s.sol

```diff
@@ -51,20 +51,39 @@ contract ForkLive is Deployer {
 
     /// @notice Forks, upgrades and tests a production network.
     /// @dev This function sets up the system to test by:
-    ///      1. reading the superchain-registry to get the contract addresses we wish to test from that network.
-    ///      2. deploying the updated OPCM and implementations of the contracts.
-    ///      3. upgrading the system using the OPCM.upgrade() function.
+    ///      0. read the environment variable to determine the
+    ///         SUPERCHAIN_OPS_ALLOCS_PATH environment variable set from superchain ops.
+    ///      1. if the environment variable is set, load the state from the given path.
+    ///      2. read the superchain-registry to get the contract addresses we wish to test from that network.
+    ///      3. deploying the updated OPCM and implementations of the contracts.
+    ///      4. upgrading the system using the OPCM.upgrade() function if not using the applied state from superchain
+    /// ops.
```

Comment:
The grammar here should be more consistent, ie. 'by: reading..., deploying'.

Reply:
Here is my proposal, assuming this is all correct:
```
    /// @dev This function sets up the system to test it as follows:
    ///      1. Check if the SUPERCHAIN_OPS_ALLOCS_PATH environment variable was set from superchain ops.
    ///      2. If set, load the state from the given path.
    ///      3. Read the superchain-registry to get the contract addresses we wish to test from that network.
    ///      4. If the environment variable wasn't set, deploy the updated OPCM and implementations of the contracts.
    ///      5. Upgrade the system using the OPCM.upgrade() function if useUpgradedFork is true.
```

Reply:
Your suggestion sounds good to me.

Reply:
https://github.com/ethereum-optimism/optimism/pull/14021/commits/802f3d489561d72389248442432ac4b04143c068